### ISSUE TITLE
Update fastapi-mailman dependencies

### ISSUE THREAD

**Issue by daniel-herrero** [marktennyson/fastapi-mailman#5]
**Is your feature request related to a problem? Please describe.**

I am currently running FastAPI 0.73.0 (with in turn uses Starlette 0.17.1), but Fastapi-Mailman [requires](https://github.com/marktennyson/fastapi-mailman/blob/master/pyproject.toml
) to run FastAPI 0.68.2 (and Starlette 0.14.2).

I believe that outdated version of Starlette make some tests to fail with the following error in the `testclient.py`:
```python
>       response = client.post("/auth/token", json=data)

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../.venv/lib/python3.10/site-packages/requests/sessions.py:577: in post

../../.venv/lib/python3.10/site-packages/requests/sessions.py:577: in post
    return self.request('POST', url, data=data, json=json, **kwargs)
../../.venv/lib/python3.10/site-packages/starlette/testclient.py:415: in request
    return super().request(
../../.venv/lib/python3.10/site-packages/requests/sessions.py:529: in request
    resp = self.send(prep, **send_kwargs)
../../.venv/lib/python3.10/site-packages/requests/sessions.py:645: in send
    r = adapter.send(request, **kwargs)
../../.venv/lib/python3.10/site-packages/starlette/testclient.py:243: in send
    raise exc from None
../../.venv/lib/python3.10/site-packages/starlette/testclient.py:240: in send
    loop.run_until_complete(self.app(scope, receive, send))
/home/daniel/.pyenv/versions/3.10.2/lib/python3.10/asyncio/base_events.py:617: in run_until_complete
    self._check_running()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <_UnixSelectorEventLoop running=False closed=False debug=False>

    def _check_running(self):
        if self.is_running():
>           raise RuntimeError('This event loop is already running')
E           RuntimeError: This event loop is already running

/home/daniel/.pyenv/versions/3.10.2/lib/python3.10/asyncio/base_events.py:577: RuntimeError
----------------------------------------------------------------------------------- Captured stderr setup ------------------------------------------------------------------------------------
Got an error creating the test database: database "test" already exists

================================================================================== short test summary info ===================================================================================
FAILED tests/auth/test_api.py::test_login_successfully - RuntimeError: This event loop is already running
```

**Describe the solution you'd like**

Try to unset the fixed versions from the dependencies (`pyproject.toml`) to evaluate if there's (or not) a real problem  when using the last available versions.

```
[tool.poetry.dependencies]
python = "^3.6.2"
fastapi = "^0.68.1"
Jinja2 = "^3.0.1"
aiosmtplib = "^1.1.6"
pydantic = "^1.8.2"
email-validator = "^1.1.3"
dnspython = "^2.1.0"
```

In case of an error, try to locate and find a fix for the conflicting code.

**Describe alternatives you've considered**

A partial solution would be to update part of the dependencies to their last unaffected versions (in an error case scenario).

**Additional context**

Using `Python 3.10.2`

**Comment by daniel-herrero** [comment]
Hi @marktennyson,

We have tried with the following dependency versions in the .toml, and the tests run with `$ poetry run tox` succeeded.
```
[tool.poetry.dependencies]
python = "^3.6.2"
fastapi = "~0"
Jinja2 = "~3"
aiosmtplib = "^1.1.6"
pydantic = "~1"
email-validator = "~1"
dnspython = "~2"
```

Please, tell me if you're interested in a PR

**Comment by marktennyson** [comment]
@daniel-herrero 
Yes, please create a pull request with the changes. 
Thanks

**Comment by daniel-herrero** [comment]
There you have. Thank you for your Fastapi extension!

**Comment by marktennyson** [comment]
Fixed at the new version => 0.0.4.
Thanks for the PR
